src/app/shared/users.mock.ts

```diff
@@ -1,11 +1,6 @@
-import { Injectable } from '@angular/core';
+import { User } from './users.model';
 
-@Injectable({
-  providedIn: 'root'
-})
-export class UsersService {
-
-  users = [
+export const UserList: User[] = [
```

Comment:
`UserList` can be renamed to `usersList`

Reply:
renamed the list name to usersList